src/lamplib/src/genny/tasks/run_tests.py

```diff
@@ -10,6 +10,20 @@
 
 SLOG = structlog.get_logger(__name__)
 
+# See the logic in _setup_resmoke.
+# These are the "Binaries" evergreen artifact URLs for mongodb-mongo compile tasks.
+# The binaries must be compatible with the version of the mongo repo checked out in use for resmoke,
+# which is the sha "6e2dcc5a39eb2de9d2e8209271115c078ae16470" mentioned below.
+CANNED_ARTIFACTS = {
+    "osx": "https://dsi-donot-remove.s3.us-west-2.amazonaws.com/compile_artifacts/mongodb-macos-x86_64-6.0.0.tgz",
+    "amazon2": "https://dsi-donot-remove.s3.us-west-2.amazonaws.com/compile_artifacts/mongodb-linux-x86_64-amazon2-6.0.0.tgz",
+    "ubuntu1804": "https://dsi-donot-remove.s3.us-west-2.amazonaws.com/compile_artifacts/mongodb-linux-x86_64-ubuntu1804-6.0.0.tgz",
+    "ubuntu2004": "https://dsi-donot-remove.s3.us-west-2.amazonaws.com/compile_artifacts/mongodb-linux-x86_64-ubuntu2004-6.0.0.tgz",
+    "rhel70": "https://dsi-donot-remove.s3.us-west-2.amazonaws.com/compile_artifacts/mongodb-linux-x86_64-rhel70-6.0.0.tgz",
+    "rhel8": "https://dsi-donot-remove.s3.us-west-2.amazonaws.com/compile_artifacts/mongodb-linux-x86_64-rhel80-6.0.0.tgz",
+}
+MONGO_COMMIT = "6e2dcc5a39eb2de9d2e8209271115c078ae16470"
```

Comment:
Is it possible to derive the commit from the binary, using `./mongod --version` or something? Super cool if not but that'd be one less config option to track